xbmc/addons/kodi-dev-kit/include/kodi/versions.h

```diff
@@ -97,7 +97,7 @@
 #define ADDON_INSTANCE_VERSION_AUDIOENCODER_DEPENDS   "c-api/addon-instance/audio_encoder.h" \
                                                       "addon-instance/AudioEncoder.h"
 
-#define ADDON_INSTANCE_VERSION_GAME                   "2.1.0"
+#define ADDON_INSTANCE_VERSION_GAME                   "3.0.0"
 #define ADDON_INSTANCE_VERSION_GAME_MIN               "2.1.0"
```

Comment:
You should also change MIN to 3.0.0

Reply:
On changing min to 3.0.0, game.libretro addon is disabled, I have done this yesterday also. This was the screenshot of the same.
![image](https://user-images.githubusercontent.com/56973333/126114886-048e456b-80e7-4f84-877b-0e99184feeae.png)

Reply:
It worked @garbear, Nick suggested me to change version of addon.xml in game.libretro also to 3.0.0 than after building game.libretro it worked fine. Thank you!!

Reply:
Good, we want old versions of game.libretro to be rejected (we require a compatible version, otherwise we get your segfault, and segfaults should be avoided). I'll fire up my windows PC later today and figure out where the game.libretro build should be.

Reply:
Hey @garbear I tried by changing version of game.libretro to 3.0.0 in the file `xbmc\cmake\addons\build\game.libretro\game.libretro\addon.xml` and then builded it again, but it also did not help.
If you will be able to figure out what to do then please guide.

Reply:
Sorry, I'm pressed hard at work and can't find a moment to spare for testing. From what I remember, the `prepare` script runs the CMake command and generates VS project files. Instead of building with `cmake --install`, open the .sln in VS, right click on the project, and compile it that way. Hunt down where it places the compiled files. Either copy them to `addons/`, or to simplify your life, make a symlink. That should cause game.libretro dependent on 3.0.0 to appear in `addons/`

Reply:
Thanks @garbear , I have not done exactly the same but done something which I think is working, First step I do was to change game version in file versions.h to 3.0.0 from game.libetro.sln (I have changed it in kodi.sln but I realize today that it was 2.1.0 here in game.libretro.sln) then I have changed individual files at kodi-build/addons/game.libretro by the files I obtained after building game.libretro.sln. It seems to work right now, and if there will any problem I will comment here later on

Reply:
Hello @garbear , On adding one new function, again I realise that same problem is coming, Kodi was searching for stale version, so I have changed game versions to 3.1.1 but it is not seem to work(game.libretro is disabled now),so is there any specific pattern to which these version should be changed?

Reply:
Yes there's a pattern, which is called "semver" or "semantic versioning". Semver defines the three numbers as major, minor, patch. Patch is no change to API, minor is backwards-compatible change, and major is backwards-incompatible change.

You can bump versions how you see fit for development, and we'll reset to 3.0 when we merge after the summer.

So you're still getting stale game.libretro? I hate that my workload is so high, I'd like to hop on windows and document how to make it not stale and always working. I'll do my best to get some documentation written after work tonight.